src/runtime/components/Table.vue

```diff
@@ -87,6 +87,11 @@ export interface TableProps<T extends TableData> extends TableOptions<T> {
    * @defaultValue false
    */
   sticky?: boolean
+  /**
+   * Whether the table should have a sticky footer.
+   * @defaultValue false
+   */
+  stickyFooter?: boolean
```

Comment:
Can't we use the `sticky` prop for that? 🤔 Something like: `sticky?: boolean | 'header' | 'footer'`